Client/Frontend/Fakespot/Views/StarRatingView.swift

```
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class StarRatingView: UIView {
+    enum UX {
+        static let starSize: CGFloat = 24
+        static let starCount = 5
+    }
+
+    var rating: Double = 0.0 {
+        didSet {
+            updateStarImages()
+        }
+    }
+
+    private lazy var stackView: UIStackView = .build { stackView in
+        stackView.axis = .horizontal
+        stackView.spacing = 0
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
```

Same as for `AdjustRatingView`: we should throw a fatal error here.